Adding workshop info (#44)
uk-tre/website: docs/events/wg_workshops/2024-03-14-march-meeting/workshop-community-governance.md

# UK TRE Community Governance and progress

**Leads**: David Sarmiento (Turing), Hari Sood (Turing)

## Proposal

### Summary

The UK TRE Community received funding as part of the [DARE UK Community grants call](https://dareuk.org.uk/funding-opportunity-dare-uk-invites-applications-for-funding-from-new-and-existing-community-groups/).

> This funding allowed for dedicated time from David (0.2 FTE) and Hari (0.5 FTE) between November and March to work on key community deliverables.

Reviewer: ```suggestion
This funding allowed for dedicated time from David (0.5 FTE) and Hari (0.5 FTE) between November and March to work on key community deliverables.
```

> The full funding application can be downloaded here - {download}`DARE UK Community Groups Funding Application - UK TRE <../../../_static/dare_uk_funding.pdf>`

Reviewer: ```suggestion
The full funding application can be downloaded here - [DARE UK Community Groups Funding Application - UK TRE](https://zenodo.org/records/10593493)
```
What if we share the one on Zenodo rather than hosting here? Also I took out full budget in the Zenodo one, I dont actually have a problem with sharing but it is our salaries after all

Author: Good idea!

In this session, we want to dive deeper into the work we've been doing, and get feedback from the community.
Specific things we'd like to cover include:

- Feedback on the new [Community website](https://github.com/uk-tre/hugo-website) and resource hub
- Feedback on the [foundational governance for the community](https://github.com/uk-tre/community-management/issues/65)
- How to transition effectively back to an unfunded community (hopefully in the short term!)

### Required preparation

None required, but reading over the links above (website and governance docs) would help the discussion.

> 

Reviewer: ```suggestion

Those that want to can also explore [our public board](https://github.com/orgs/uk-tre/projects/1/views/1) that reflect the work done over this period
```
